### ISSUE TITLE
[REQ] back button

### ISSUE THREAD

**Issue by wschoot** [torandi/garmin-wrapped#6]
Is it possible to implement a back-button? As far as I've found out, I can only move forward (using a single left mouseclick -- no keyboard support)

Can we add a gesture for moving one slide back?

**Comment by torandi** [comment]
Yeah, I've thought the same thing. Need to figure out a good input for navigating back.

But maybe just adding left/right arrow support for back and forward might work.
